packages/land/contracts/polygon/PolygonLandStorageMixin.sol

```diff
@@ -20,7 +21,13 @@ import {IOperatorFilterRegistry} from "../interfaces/IOperatorFilterRegistry.sol
 //│         PolygonLand         │     _trustedForwarder     │     107      │   0    │                    t_address                     │  0  │ /build-info/3abb06944792151ded64cbcd19543bb1.json │      20       │
 //│         PolygonLand         │  operatorFilterRegistry   │     108      │   0    │     t_contract(IOperatorFilterRegistry)3942      │  0  │ /build-info/3abb06944792151ded64cbcd19543bb1.json │      20       │
 
-contract PolygonLandStorageMixin is ContextUpgradeable {
+contract PolygonLandStorageMixin is Initializable, Context {
+    // To avoid storage slot conflicts during upgrades, we add these variables.
+    // variables naming to be  upgraded
+    uint8 private _initialized;
+    bool private _initializing;
+    uint256[50] private ___gap;
+
```

Comment:
Why did you add this variables ?

Reply:
@adjisb

Reply:
This was expected, it is ok.

Reply:
`After 5.0, the storage location of some variables were changed. This is the case for Initializable and all the upgradeable contracts since they now use namespaced storaged locations.`
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/CHANGELOG.md